lab2.js

```diff
@@ -122,24 +181,49 @@ assert(human.sayHello(klingon) === 'nuqneH',
 //*********************************************************
 function max(array) {
   // TODO: return the largest number in the given array
+  return Math.max.apply(Math, array);
```

Comment:
glad to see you playing with apply. This isn't the case where you'd usually use it, but it works.